lib/tbd/geo.rb

```diff
@@ -729,14 +745,7 @@ def resetKIVA(model = nil, boundary = "Foundation")
       kiva.remove
     end
 
-    # Reset surface KIVA-related objects.
-    model.getSurfaces.each do |surface|
-      surface.resetAdjacentFoundation
-      surface.resetSurfacePropertyExposedFoundationPerimeter
-
-      b = surface.outsideBoundaryCondition.capitalize
-      surface.setOutsideBoundaryCondition(boundary) unless b == boundary
-    end
+    log(INF, "Purged KIVA objects from model (#{mth})") if kva
```

Comment:
If successful, log _informative_ message that previous KIVA objects have been purged. This makes its way to the measure runner (e.g. on-screen feedback), as well as generated "tbd.out.json" file. Worth raising awareness as it's a significant change to a model.